lib/services/prepare-platform-js-service.ts

```diff
@@ -36,6 +36,7 @@ export class PreparePlatformJSService extends PreparePlatformService implements
 	public async preparePlatform(config: IPreparePlatformJSInfo): Promise<void> {
 		if (!config.changesInfo || config.changesInfo.appFilesChanged || config.changesInfo.changesRequirePrepare) {
 			await this.copyAppFiles(config);
+			this.copyAppResourcesFiles(config);
```

Comment:
I assume this is related to https://github.com/NativeScript/nativescript-cli/pull/3329/files#diff-d7315453ac31262b45d0c2530d69cd94R47 change. Can you describe a little bit more what is the idea